demo/flutter/integration_test/app_test.dart

```diff
@@ -0,0 +1,128 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+import 'dart:typed_data';
+
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:wav/wav.dart';
+
+import 'package:porcupine_flutter/porcupine.dart';
+import 'package:porcupine_flutter/porcupine_error.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  final String accessKey = "{TESTING_ACCESS_KEY_HERE}";
+  final String platform = Platform.isAndroid ? "android" : Platform.isIOS ? "ios" : "error";
+
+  Future<List<int>> loadAudioFile(String audioPath) async {
+      const INT16_MAX = 32767;
+      const INT16_MIN = -32768;
+
+      var audioFileData = await rootBundle.load(audioPath);
+      Wav audioFile = await Wav.read(audioFileData.buffer.asUint8List());
+      List<int> pcm = audioFile.channels[0].map((f) {
+          var i = (f * INT16_MAX).truncate();
+          if (f > INT16_MAX) i = INT16_MAX;
+          if (f < INT16_MIN) i = INT16_MIN;
+          return i;
+        }).toList();
+    return pcm;
```

Comment:
why don't we load the wav file as a 16-bit pcm?  The wav dart package claims that it can read 16-bit ints

Reply:
I think that it meant the library can read WAV files that arestored in the Int16 format. The package only presents the one interface to get the PCM and it's in Float32 format.

Reply:
ah, I see. That's unfortunate.